src/Resources/config/services.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+        https://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <!-- Controllers -->
+        <service id="karo-io.controller.dashboard" class="KaroIO\MessengerMonitorBundle\Controller\DashboardController">
+            <tag name="controller.service_arguments" />
+            <argument type="service" id="twig" />
+            <argument type="service" id="karo-io.messenger_monitor.receiver_locator" />
+            <argument type="service" id="karo-io.messenger_monitor.failed_message.repository" />
+        </service>
+
+        <service id="karo-io.controller.retry_failed_message" class="KaroIO\MessengerMonitorBundle\Controller\RetryFailedMessageController">
+            <tag name="controller.service_arguments" />
+            <argument type="service" id="karo-io.messenger_monitor.failed_message.retry" />
+            <argument type="service" id="session" />
+            <argument type="service" id="router" />
+        </service>
+
+        <service id="karo-io.controller.reject_failed_message" class="KaroIO\MessengerMonitorBundle\Controller\RejectFailedMessageController">
+            <tag name="controller.service_arguments" />
```

Comment:
I don't think we need this tag on any of the controllers because you're using proper DI (as you should, because this is a 3rd party lib and there is only 1 action per controller) and not relying on action injection.